build/release/teamcity-mark-build-qualified.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
We usually add a line like this at the top of the `teamcity-*` scripts. Could we add it here too?

```
set -euxo pipefail
```